### DISCUSSION TITLE
Far too many of our users are finding themselves unable to login with error shown

### DISCUSSION THREAD

**Discussion by rachellawson** [GitbookIO/community#328] [0. General bugs and questions] [answered]
Is it just our organisation that is suffering from lots of our users finding themselves unable to login, with the following error?

![image](https://github.com/GitbookIO/community/assets/807248/9251141b-79bc-431e-98da-faf1bc806fdf)

It is affecting way too many of our users now and having a detrimental effect upon confidence in the platform.

Can we get a full explanation of what’s happening?

**Comment by aikoii** [comment]
Hey @rachellawson ! I'm sorry you're experiencing this issue. Let me check with our team. In the meantime, could you let me know for which users this is happening?

**Reply by rachellawson** [reply]
We have had this happen for over a dozen users in the govstack.gitbook.io org so far. Including me. It just keeps happening.

**Reply by jpreynat** [reply]
Hi @rachellawson, can you provide more information on which screen, space, and if possible the URLs you are seeing this error message on?
If you are able to, can you also send a screenshot of your browser's console when this happens. It should show some red lines when this is happening, and it would help us understand your issue.

Thanks in advance 🙏

**Reply by rachellawson** [reply]
Hi. The error message is attached above.

Your support team have fixed a whole bunch of these occurrences - I don’t want to post email addresses here but they are all occurring when accessing spaces with the org ID `pxmRWOPoaU8fUAbbcrus`.

not seen any red lines in any screenshots sent to me - maybe they are in the users’ emails?

**Reply by jpreynat** [reply]
Hi.

So I've tried to access most of the spaces in your organization with your account, and everything is working fine on my end.

Can you try these two things:
1. Clear your browser's cache for app.gitbook.com.
You can find the instructions by browser on this page: https://community.qlik.com/t5/Official-Support-Articles/How-to-Clear-All-Browser-Cache-or-for-specific-site/ta-p/1710298
You will have to login again to the application when it's done.
2. If you still get the same error message after step 1., copy the URL of your browser when the error happens and send it to me.
As an additional bonus, if the error pops up again, try to open your browser's console and copy/screenshot the error that you see there.
Here's a guide to open your browser's console: https://balsamiq.com/support/faqs/browserconsole/#google-chrome

Thanks again

**Reply by rachellawson** [reply]
@jpreynat yes, we have had many users get this error at login. Clearing the browser cache does not resolve the issue and they have had to raise support calls via email for them the be resolved.

It’s not how to fix a single user’s issue - this is about why so many users are having this issue.

**Reply by taszap** [reply]
Hi @rachellawson we are trying to pin down the issue but need some clarification please. Are those users experiencing this error right now? Or it happens occasionally? 

Basically we need a specific examples of users who are having this issues so we can identify the errors and prevent this situation from happening again.

**Reply by rachellawson** [reply]
Once it happens to someone, they are locked out until they get support from you. 

Right now, Ain (WXd2SEYKi1Wg7ebQT8w0iLjOE6x2) is locked out

**Reply by taszap** [reply]
Hi @rachellawson, thanks but that's a different case. If the user is logging in with wrong email they will not have access to content so that's to be expected. In Ain's case he is not experiencing error but simply needs to be invited to the org with new email (solved). 

For example, if I have two emails: nataszapersonal@gmail.com and natasza@gitbook.com (member of GitBook org) - I will not be able to access GitBook's organization with nataszapersonal@gmail.com unless this email is also added as a member. 

Sadly to investigate this further we need specific users who are logging in using correct email and experiencing something went wrong error. You can email them at support@gitbook.com for privacy reasons - thank you!

**Reply by rachellawson** [reply]
Yeah, sorry, I only just found that out, too. 

Taylor Downs has suffered with it this week - SYDio7XH4ASnjkHgXnsXaVzN4OS2

**Reply by aikoii** [reply]
@rachellawson it looks like the issue has been resolved for Taylor. Let us know if anyone else is having trouble accessing GitBook. Thanks!

**Reply by rachellawson** [reply]
Yes, I know it has been resolved for Taylor. It has been resolved for many people. 

The whole point of the discussion isn't about resolving an individual user but to understand _why_ this is happening again and again. 
What is going on? Can you give a technical explanation as to why so many of my users are getting locked out in this way? It is damaging confidence.

**Reply by taszap** [reply]
Hi Rachel, thanks for sharing Taylor's example. It helped us narrow down the issue and will run a fix now which should prevent this happening in the future. I will share more details over email!

**Reply by rachellawson** [reply]
Ah that's awesome! I'm glad we will have something to explain what happened. 
Seriously, it means a lot to be able to explain why things were happening.

**Reply by rachellawson** [reply]
Thanks so much for the email with a full explanation of what was causing the issue. It was genuinely interesting and helps me to rebuild confidence back with our users and leadership.

I recommend you also share it here - it is always good to show _why_ things have happened and how they have been mitigated against in the future.

Good job, all!

**Reply by anvidocs** [reply]
We are experiencing the same thing! I have reached out to Gitbook support and am still waiting on a response. What was the response over email and is there a resolution yet?

**Reply by aikoii** [reply]
Hey, @anvidocs! Apologies for the wait! I'm looking into it 🙏 Also, I don't think your case is similar to what Rachel experienced

**Comment by taszap** [comment, marked as answer]
Sharing the response I have shared with Rachel for transparency (the issue is resolved)

> Thanks for your patience while we investigated. By sharing Taylor's example with us, you helped us pick up the error that was causing this.
> 
> Tl:dr
> We had a rare issue which is some cases led to errors on the computation of those permissions, leading to all the members in the group to have invalid permissions (potentially all of them) not able to access the app. When changes are made to permissions in the org, on some conditions, this triggers a computation of the permissions for multiple members in the org (new team created, or team receives added permissions, etc…) which is why it affected many users in your org.
> 
> We shipped a fix on Monday [4th of September] for this but this didn't necessarily recompute all the permissions for all your users which is why you still could have experienced issues.
> 
> My colleague has just recomputed the access for all your users which means we should not hopefully have this problem moving forward.
> 
> Please let me know if you have any additional questions.

**Comment by sidnb13** [comment]
Still experiencing this issue, but I am not part of a team/organization and am on the free tier. Any solutions?

**Reply by aikoii** [reply]
Hey @sidnb13, I'm sorry for the trouble you've been experiencing 🙏. There could be multiple reasons why you're encountering this issue. Most of the time, clearing the cookies and local data in your browser can be helpful.

Could you send a HAR file for us the next time the error pops up and  describe the exact steps you took leading up to the error? A screen recording would be super helpful. Please feel free to send us the information at support@gitbook.com.

**Comment by taszap** [comment]
Hi everyone!

As this thread evolved to report of many cases where the user is locked out, which can be unrelated I will lock this discussion. 
If you experience issues we encourage you to contact us at support@gitbook.com with more information. This will ensure you get timely help.
